box-share-sdk/src/main/java/com/box/androidsdk/share/vm/InviteCollaboratorsDataWrapper.java

```diff
@@ -0,0 +1,40 @@
+package com.box.androidsdk.share.vm;
+
+/**
+ * A special extension of DataWrapper to hold data for inviting a new Collaborator
+ */
+public class InviteCollaboratorsDataWrapper extends DataWrapper<String> {
+
+    private boolean mInvitationFailed;
+
+    public InviteCollaboratorsDataWrapper(String data, int strCode, boolean invitationFailed) {
+        this.mData = data;
+        this.mStrRes = strCode;
+        this.mInvitationFailed = invitationFailed;
+    }
+
+    /**
+     * Returns true if all invitations succeeded.
+     * @return true if all invitations succeeded
+     */
+    @Override
+    public boolean isSuccess() {
+        return !isInvitationFailed();
```

Comment:
Is mData != null not sufficient?

Reply:
It is not because the data will be there even for failure cases. For instance, the data might hold the name of collaborator that was not invited.